file_store/src/error.rs

```diff
@@ -24,6 +22,12 @@ pub enum Error {
     Config(#[from] config::ConfigError),
     #[error("mpsc channel error")]
     Channel,
+    #[error("unsupported datarate {0}")]
+    UnsupportedDataRate(String),
+    #[error("unsupported invalid_reason {0}")]
```

Comment:
may want to make this more explicit.. reason for what?

Reply:
[c688769](https://github.com/helium/oracles/pull/215/commits/c688769c8b664bac808b180d82c0e7adb3cdd09f)